src/main/java/frc/robot/RobotContainer.java

```diff
@@ -246,12 +246,16 @@ private void configureDriverBindings() {
     driverController
         .rightTrigger()
         .whileTrue(
-            new AutoAimCommand(
-                    driveSubsystem,
-                    FieldPositions.getInstance().getSpeakerTranslation(),
-                    () -> getJoystickInput(driverController, JoystickAxis.Ly),
-                    () -> getJoystickInput(driverController, JoystickAxis.Lx))
-                .repeatedly());
+            new ParallelCommandGroup(
+                new AutoAimCommand(
+                        driveSubsystem,
+                        FieldPositions.getInstance().getSpeakerTranslation(),
+                        () -> getJoystickInput(driverController, JoystickAxis.Ly),
+                        () -> getJoystickInput(driverController, JoystickAxis.Lx))
+                    .repeatedly(),
+                manipulatorSubsystem
+                    .getShooterAndArmPrepareCommand(() -> getDistanceToSpeaker())
+                    .repeatedly()));
```

Comment:
Let's use method chaining where we can.

```suggestion
            new AutoAimCommand(
                    driveSubsystem,
                    FieldPositions.getInstance().getSpeakerTranslation(),
                    () -> getJoystickInput(driverController, JoystickAxis.Ly),
                    () -> getJoystickInput(driverController, JoystickAxis.Lx))
                .repeatedly()
                .alongWith(
                    manipulatorSubsystem
                        .getShooterAndArmPrepareCommand(this::getDistanceToSpeaker)
                        .repeatedly()));
```